### PR TITLE
Wednesday pull request

### PR DESCRIPTION
**Pull request by mikejuli**

### CODE REVIEW

File: client/src/BaseForm.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Month from './Month.jsx';
+class BaseForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+
```
Comment (jinyeong-park) [inline client/src/BaseForm.jsx:7]: If you are not planning to use a state, you can change a functional component!

File: client/src/Day.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    this.props.changeDate(r);
+
+
+
+
+    //toggle if once was clicked set second date
+  }
+
+  render() {
+    var c;
```
Comment (jinyeong-park) [inline client/src/Day.jsx:28]: If you are not using this variable, you could delete it ;-)

**Review by jinyeong-park — APPROVED** [review]

File: client/src/Day.jsx (hunk 2 above)
Comment (mikejuli) [reply client/src/Day.jsx:28]: Done it!

File: client/src/BaseForm.jsx (hunk 1 above)
Comment (mikejuli) [reply client/src/BaseForm.jsx:7]: Done it!

**Pull request merged**